src/raw/_Raw.jsx

```diff
@@ -36,7 +41,7 @@ const Raw = ({ indexPath, contentConfig, location }) => {
   if (route) {
     const { componentClass: ComponentClass, props } = flattenedRouteConfig[route].component.default;
     return (
-      <main>
+      <main className={cx('main')}>
```

Comment:
Might add a `role="main"` on this element as well, I think older browsers need that. Other than that, looks good to me.